core/test/scenarios/api-test-pptr.js

```diff
@@ -180,10 +180,6 @@ describe('Individual modes API', function() {
         .map((r) => ({url: r.url, sessionTargetType: r.sessionTargetType}))
         // @ts-expect-error
         .sort((a, b) => a.url.localeCompare(b.url));
-      expect(networkRequests).toHaveLength(10);
-      expect(networkRequests.filter(r => r.sessionTargetType === 'page')).toHaveLength(2);
-      expect(networkRequests.filter(r => r.sessionTargetType === 'iframe')).toHaveLength(2);
-      expect(networkRequests.filter(r => r.sessionTargetType === 'worker')).toHaveLength(6);
```

Comment:
All of these checks are redundant with the snapshot check just below. The snapshot check provides better details on failure so we should just fall to that if a request is missing.